addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,160 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
+    assert.expect(3);
+
+    this.data['res.partner'].records.push({ id: 100, email: 'john@jane.be', display_name: 'John Jane' });
+    this.data['res.fake'].records.push({
+        id: 10,
+        email_cc: "john@test.be",
+        partner_ids: [100],
+    });
+    await this.start ();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Should display a list of suggested recipients after opening the composer from 'Send message' button"
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        3,
+        "There should be 3 suggested recipients"
+    );
```

Comment:
Haven't run this test locally, but at a glance it makes no sense to have 3 suggested recipients based on the data. I expect to see 2 at most... (John Jane and John Test)

Reply:
The author is also used. See https://github.com/odoo/odoo/pull/59989/files#diff-7628187dfd0a399757052622138ad6eca27bca67f06e239542e237cbc63652eaR1524

Reply:
This fake model does too much. Actually, I'm not fond of fake model having some black magic with suggested recipients.

Pretty sure I'm calling a quarrel again with @seb-odoo, but test should be natural to read and understand, with no hidden stuffs. For this, code should be properly designed, with good split of behaviour and data. Suggested recipients are and should be mostly data, even the ones without any partner. I personally don't care that server doesn't do that, if it feels right I'm in favour of not following server logic.

Please take a step back and observe the huge technical debt we introduce with this bad code in test, while also discouraging fixing the server code. Creating a task is not a solution for me, because I foresee it ending up in the scrap column like many other existing tasks.

Reply:
For this one I don't care too much, especially since this is a fake model in the first place. I would argue it is better to have fake model test specific since they don't exist in the server anyway, it doesn't have place in the mock server.

(If we were testing actual `project.task` though then yes for me this should have its place.).